pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -350,6 +350,6 @@ func (m *podContainerManagerNoop) GetPodCgroupConfig(_ *v1.Pod, _ v1.ResourceNam
 	return nil, nil
```

Comment:
Can GetPodCgroupConfig also benefit from libcontainer? 
https://github.com/opencontainers/runc/blob/3d07cbc9a1adfb784eec3af9bf235a244178ece0/libcontainer/cgroups/cgroups.go#L64-L65
Or did I misunderstand it.. perhaps fails for non-systemd cgroup-driver?

Reply:
I was thinking about this as well, just felt a bit out-of-scope for this PR, but I've added it now :)

In general, I think that a lot could be improved in the cgroup area. Perhaps I can create separate issues and improve this part as a follow-up (FYI @SergeyKanzhelev).